### ISSUE TITLE
reset_simulators install only latest iOS simulators

### ISSUE THREAD

**Issue by sergii-frost** [KrauseFx/snapshot#191]
Hi, this may be not an issue but lack of my knowledge of the tool, please do not judge. 
My Xcode 6.4 has Simulators 7.1, 8.2, 8.2, 8.3, 8.4 installed. 
But hewn when I am running `snapshot reset_simulators` I am getting simulators only for latest 8.4.
Is is possible to specify somehow to configure all the simulators?

**Comment by KrauseFx** [comment]
I haven't yet looked into creating the simulators for multiple versions. Right now it only supports the currently used version from `xcode-select`.

**Comment by sergii-frost** [comment]
@KrauseFx thanks for your quick response. 
I thought I was blocked by this issue, but then found that it is possible to create additional simulators from  "Organiser" in Xcode. Feel free to mark this issue as Enhancement. It will be really nice-to-have.

**Comment by KrauseFx** [comment]
:+1:

**Comment by KrauseFx** [comment]
Solved with https://github.com/KrauseFx/snapshot/pull/192, this will be in the next release :+1:

**Comment by sergii-frost** [comment]
@KrauseFx Perfect! Thank you!

**Comment by fastlanebot** [comment]
This issue was migrated to https://github.com/fastlane/fastlane/issues/2611. Please post all further comments there.

`fastlane` is now a mono repo, you can read more about the change in our [blog post](https://krausefx.com/blog/our-goal-to-unify-fastlane-tools). All tools are now available in the [fastlane main repo](https://github.com/fastlane/fastlane) :rocket:
